apps/schools/domains/student/components/currentStudent/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import { Col, Row, Typography } from 'antd'
+import Image from 'next/image'
+
+import { useGetStudentQuery } from '@domains/organization/redux/organizationApi'
+import { getUuidFromUrl } from '@domains/common/utils/getUuidFromUrl'
+
+import EmptyWrapper from '@domains/common/components/containers/EmptyWrapper'
+import styles from './styles/styles.module.scss'
+import { ErrorType } from '@store/commonApi'
+import router from 'next/router'
+import duckEmptyPage from '@public/image/duckEmptyPage.svg'
+import { Button } from '@domains/common/components/button'
+import { Field } from '@domains/common/components/field'
+
+const CurrentStudent = () => {
+    const [isModalVisible, setIsModalVisible] = useState(false)
+    // const [mutation, isDeleteFinished] = useDeleteCircleMutation()
```

Comment:
why is there a comment here?

Reply:
I will add this logic in the next PR with change and delete of student